tests/api/test_user_profile.py

```diff
@@ -0,0 +1,396 @@
+        data = response.json()
+
+        # Verify all config sections are present and populated
+        assert "sectors" in data and len(data["sectors"]) > 0
+        assert "sector_roles" in data and len(data["sector_roles"]) > 0
+        assert "countries" in data and len(data["countries"]) > 0
+        assert "languages" in data and len(data["languages"]) > 0
+        assert (
+            "gis_expertise_levels" in data
+            and len(data["gis_expertise_levels"]) > 0
+        )
+
+        # Verify data matches our configs
+        assert data["sectors"] == SECTORS
+        assert data["sector_roles"] == SECTOR_ROLES
+        assert data["countries"] == COUNTRIES
+        assert data["languages"] == LANGUAGES
+        assert data["gis_expertise_levels"] == GIS_EXPERTISE_LEVELS
+
+    @pytest.mark.asyncio
+    async def test_profile_config_no_auth_required(self, client):
+        """Test that profile config endpoint doesn't require authentication."""
+        response = await client.get("/api/profile/config")
+        assert response.status_code == 200
+
+
+class TestUserProfileAPI:
+    """Test the user profile update API endpoint."""
+
+    def setup_method(self):
+        """Set up test data."""
+        # Get valid values from configs for testing
+        self.valid_sector = next(iter(SECTORS.keys()))
+        self.valid_role = next(iter(SECTOR_ROLES[self.valid_sector].keys()))
+        self.valid_country = next(iter(COUNTRIES.keys()))
+        self.valid_language = next(iter(LANGUAGES.keys()))
+        self.valid_expertise = next(iter(GIS_EXPERTISE_LEVELS.keys()))
+
+    @pytest.mark.asyncio
+    async def test_update_profile_requires_auth(self, client):
+        """Test PATCH /api/auth/profile requires authentication."""
+        response = await client.patch(
+            "/api/auth/profile", json={"first_name": "John"}
+        )
+        assert response.status_code == 401
+        assert "Missing Bearer token" in response.json()["detail"]
+
+    @pytest.mark.asyncio
+    async def test_update_profile_success_basic_fields(
+        self, client, user, auth_override
+    ):
+        """Test successful profile update with basic fields."""
+        auth_override(user.id)
+
+        update_data = {
+            "first_name": "John",
+            "last_name": "Doe",
+            "profile_description": "Forest researcher interested in conservation",
+        }
+
+        response = await client.patch("/api/auth/profile", json=update_data)
+        assert response.status_code == 200
+
+        data = response.json()
+        assert data["firstName"] == "John"
+        assert data["lastName"] == "Doe"
+        assert (
+            data["profileDescription"]
+            == "Forest researcher interested in conservation"
+        )
+        assert data["id"] == user.id
+        assert data["name"] == user.name  # Original fields preserved
+
+    @pytest.mark.asyncio
+    async def test_update_profile_success_detailed_fields(
```

Comment:
the `test_profile_fields_roundtrip_auth_me` is doing the same as this test function, but also checking the response of auth/me. I would delete this function and keep only `test_profile_fields_roundtrip_auth_me`.